nsxt/resource_nsxt_cluster_virtual_ip.go

```diff
@@ -0,0 +1,114 @@
+/* Copyright © 2023 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt-mp/nsx/cluster"
+	nsxModel "github.com/vmware/vsphere-automation-sdk-go/services/nsxt-mp/nsx/model"
+)
+
+var ClusterVirtualIPForceType = []string{
+	nsxModel.ClusterVirtualIpProperties_FORCE_TRUE,
+	nsxModel.ClusterVirtualIpProperties_FORCE_FALSE,
+}
+
+var DefaultIPv4VirtualAddress = "0.0.0.0"
+
+var DefaultIPv6VirtualAddress = "::"
+
+func resourceNsxtClusterVirualIP() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtClusterVirualIPCreate,
+		Read:   resourceNsxtClusterVirualIPRead,
+		Update: resourceNsxtClusterVirualIPUpdate,
+		Delete: resourceNsxtClusterVirualIPDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"force": {
+				Type:         schema.TypeString,
```

Comment:
Any reason not to make this `TypeBool`?

Reply:
In the client function, the type of parameter force is *string is  https://github.com/vmware/vsphere-automation-sdk-go/blob/master/services/nsxt-mp/nsx/cluster/ApiVirtualIpClient.go

Reply:
`Setvirtualip(forceParam *string, ip6AddressParam *string, ipAddressParam *string)`

Reply:
We can still make this `TypeBool` for the user

Reply:
done